management/nodes_test.go

```diff
@@ -43,6 +44,66 @@ func TestNodeRegister(t *testing.T) {
 			}
 		})
 
+		t.Run("Reregister with different node name (no force - should fail)", func(t *testing.T) {
```

Comment:
actually you re-register with the same node name in the test.

Reply:
I think here should be 4 tests.
Same name - reregister = false
Same name - reregister = true
Same address-region - reregister = false
Same address-region - reregister = true

Reply:
Added all tests